src/add-user/add-user.tsx

```diff
@@ -0,0 +1,76 @@
+import React, {useState} from 'react';
+import {Alert, Button, Text, TextInput, View} from 'react-native';
+import {
+  validateBirthDate,
+  validateEmail,
+  validateName,
+  validatePassword,
+  validatePhone,
+} from '../utils/validation';
+
+export const AddUserScreen = () => {
```

Comment:
booa! tenta sempre manter o mesmo sufixo para as telas. no seu caso, como vc usou o sufixo `Screen`, vai ser sempre `HomeScreen`, `AddUserScreen` e assim por diante 👍

Reply:
A única coisa que faltou aqui é a tipagem do componente
```suggestion
export const AddUserScreen: React.FC = () => {
```